Library/Contributions/brew_bash_completion.sh

```diff
@@ -48,6 +48,8 @@ __brewcomp ()
 
     for s in $1; do
         __brewcomp_words_include "$s" && continue
+        # do not auto-complete "instal" abbreviation for "install"
+        [[ $s == instal ]] && continue
         list="$list$s$sep"
```

Comment:
Have a look at the `bash` man page in the [QUOTING section](http://manpages.ubuntu.com/manpages/vivid/en/man1/bash.1.html#contenttoc11). The relevant stuff starts with “Words of the form $'string' are treated specially.”